torchci/package.json

```diff
@@ -75,5 +76,9 @@
     "nock": "^13.2.6",
     "prettier": "2.6.2",
     "typescript": "4.6.3"
+  },
+  "browser": {
```

Comment:
I'm a bit confuse on why Vercel deployment starts to fail with some missing modules: https://vercel.com/fbopensource/torchci/HgpkArjaQdpGtLuPhX4kf7gk3cx3 when running `yarn run build`.  This could also be reproduced locally.  The fix according to https://bobbyhadz.com/blog/module-not-found-cant-resolve-fs is to turn these off, but I still don't know why yet

Reply:
I will dig a bit deeper into the change here in `package.json`, it works but the why is missing

Reply:
I figure this one out.  The most important thing is that opensearch library cannot be used in client-side browser https://github.com/elastic/elasticsearch-js#browser, so putting the code in `jobUtils` which is imported by several HUD pages is a bad choice.  So I refactor the code to put `hasSimilarFailures` into `drciUtils` instead.  That module is used only by Dr.CI API and bot, which is server-side.